### PR TITLE
update minuteman to fixed

### PR DESCRIPTION
**Pull request by aeyakovenko**

### CODE REVIEW

File: apps/navstar/src/navstar_app.erl [hunk 1]
```diff
@@ -39,7 +39,7 @@ maybe_start_minuteman() ->
         false ->
             ok;
         true ->
-            {ok, _} = application:ensure_all_started(minuteman)
+            {ok, _} = application:ensure_all_started(minuteman, permanent)
```
Comment (aeyakovenko) [inline apps/navstar/src/navstar_app.erl:42]: @drewkerrigan is this possible to test from CT?

**Review by GoelDeepak — COMMENTED** [review]
lgtm. I believe you would squash the changes

File: apps/navstar/src/navstar_app.erl (hunk 1 above)
Comment (aeyakovenko) [reply apps/navstar/src/navstar_app.erl:42]: so this works, but kills the entire node as expected, so we cant write a simple CT test
```
$ ./rebar3 as test ct --readable=false --suite=test/navstar_SUITE.erl
===> Verifying dependencies...
===> Compiling navstar_rest
===> Compiling navstar_dns
===> Compiling navstar_overlay
===> Compiling navstar
===> Running Common Test suites...
15:42:40.006 [info] Application lager started on node nonode@nohost
15:42:40.006 [info] Application riak_dt started on node nonode@nohost
Setup running ...
Directories verified. Res = {[ok],[]}
Setup finished processing hooks ...
15:42:40.022 [info] Application setup started on node nonode@nohost
15:42:40.082 [info] Starting reporters with []
15:42:40.082 [info] Application exometer_core started on node nonode@nohost
15:42:40.430 [info] Application mnesia started on node nonode@nohost
15:42:40.459 [info] Application lashup started on node nonode@nohost
15:42:40.461 [info] Application jsx started on node nonode@nohost
15:42:40.464 [info] Application cowlib started on node nonode@nohost
15:42:40.484 [info] Application ranch started on node nonode@nohost
15:42:40.491 [info] Application cowboy started on node nonode@nohost
15:42:40.546 [info] Did not load config: "/opt/mesosphere/etc/navstar.app.config"
15:42:40.563 [info] Did not load config: "/opt/mesosphere/etc/navstar.app.config"
15:42:40.578 [info] Did not load config: "/opt/mesosphere/etc/navstar.app.config"
15:42:40.670 [warning] Overlay Poller could not poll: {failed_connect,[{to_address,{"10.0.79.182",5051}},{inet,[inet],econnrefused}]}
15:42:40.697 [info] Couldn't find config directory: "/opt/mesosphere/etc/minuteman.config.d"
15:42:40.749 [warning] Terminating, due to: badarg, in state: idle, with state data: {state,#Port<0.189289>,26,8875,0,undefined,[],undefined,0}
15:42:40.768 [warning] Received unknown info: {'EXIT',<0.841.0>,shutdown}, in state: {state,<0.772.0>,#Ref<0.0.2.978>,[],[]}
{"Kernel pid terminated",application_controller,"{application_start_failure,minuteman,{{shutdown,{failed_to_start_child,minuteman_network_sup,{shutdown,{failed_to_start_child,minuteman_lb_mgr,badarg}}}},{minuteman_app,start,[normal,[]]}}}"}
```

**Review by GoelDeepak — APPROVED** [review]
lgtm

**Pull request merged**